scripts/tcl_commands/all.tcl

```diff
@@ -681,7 +709,14 @@ proc prep {args} {
             } else {
                 set drc_exclude_list "$::env(DRC_EXCLUDE_CELL_LIST)"
             }
-            gen_exclude_list -lib resizer_opt -drc_exclude_list $drc_exclude_list -output $::env(synthesis_tmpfiles)/resizer_opt.exclude.list -drc_exclude_only -create_dont_use_list
+
+            puts_verbose $drc_exclude_list
+
+            gen_exclude_list \
+                -lib resizer_opt \
+                -drc_exclude_list $drc_exclude_list \
+                -drc_exclude_only \
```

Comment:
why is it `-drc_exlucde_only`? probably outside of this pr scope though

Reply:
I have no earthly idea